Libraries/Network/__tests__/FormData-test.js

```diff
@@ -55,4 +55,33 @@ describe('FormData', function () {
     };
     expect(formData.getParts()[0]).toMatchObject(expectedPart);
   });
+
+  it('should return values based on the given key', function () {
+    formData.append('username', 'Chris');
+    formData.append('username', 'Bob');
+
+    expect(formData.getAll('username').length).toBe(2);
```

Comment:
Nit: We could add a no-match test, where we get an empty array, otherwise this looks good to me

Reply:
Added under #d176edbc681